src/App.js

```diff
@@ -29,6 +29,16 @@ function App() {
     console.log(event.target.lname.value);
     console.log(event.target.phone.value);
 
+    fetch('http://localhost:3001/', {
+      method: 'POST',
+      body: JSON.stringify({firstName: event.target.fname.value, lastName: event.target.lname.value, phone: event.target.phone.value}),
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    })
+      .then(response => response.json())
```

Comment:
This is probably fine for now, but note that this is erroring out.

response.json() attemps to parse the string that comes back as a json object, but it really is just "hello world" (the response from the server isn't valid JSON, but we're attempting to parse it as such)

Reply:
parsing JSON means parsing a string that follows a specification (structure) such that it can be turned into a javascript object.